src/index.html

```diff
@@ -7,12 +7,69 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          class="card__image-detail"
+          src="./images/imac.jpeg"
+          alt="IMac image"
+        />
+      </div>
+
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
```

Comment:
Use semantic tags where possible
<img width="966" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/7b5e69d8-d9f9-4cd6-baa7-45b9d07ea980">

It looks like a title, so you can use some heading such as `h2/h3`. For other elements you can use semantic tags as well instead of divx